target/linux/ramips/dts/mt7621_xiaoyu_xy-c5.dts

```diff
@@ -0,0 +1,103 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "xiaoyu,xy-c5", "mediatek,mt7621-soc";
+	model = "XiaoYu XY-C5";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
```

Comment:
@rushxrushx 

> I start this job modified from PBR-M1 and many issue come from it :)

That raises a question here: Is your router also using 115200 as the UART baudrate or you just copied this one from PBR-M1 as well? Not many vendors bother to change this from 57600.

Reply:
57600 at uboot, Do I need change this to 57600?

Reply:
@981213 
uart baudrate is set to 57600 now. (same as uboot)